config/deploy/cap-dev.rb

```diff
@@ -1,5 +1,5 @@
 # see https://github.com/sul-dlss/sul_pub/wiki/Servers-Deployment-environment
-server 'sul-pub-cap-dev.stanford.edu', user: 'pub', roles: %w(web db app)
+server 'sul-pub-cap-dev-a.stanford.edu', user: 'pub', roles: %w(web db app)
```

Comment:
Old dev-a also had the harvester_dev role. Should this also?

Reply:
yes, I added that ... thanks